dropwizard-validation/src/test/java/io/dropwizard/validation/SelfValidationTest.java

```diff
@@ -55,10 +58,12 @@ protected void append(ILoggingEvent iLoggingEvent) {
     @BeforeAll
     static void setUp() {
         LoggerContext loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
-        Logger root = loggerContext.getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME);
-        listAppender.setContext(root.getLoggerContext());
+        Logger selfValidatingValidatorLogger = loggerContext.getLogger(SelfValidatingValidator.class);
+        selfValidatingValidatorLogger.setAdditive(false);
+        selfValidatingValidatorLogger.setLevel(Level.DEBUG);
```

Comment:
Logback defines the `DEBUG` level by default, but our appender can only process events with level `INFO` or higher. We _could_ change the level, but I don't have a strong opinion on either version.

Reply:
No strong opinions either - as you deduced I was picking up on the logback default, but maybe we can shave some microseconds off the test run by upping the log level to INFO. I'll do that.